build-system/amp.extern.js

```diff
@@ -287,6 +287,9 @@ ReactDatesConstants.ANCHOR_LEFT;
 /** @const {string} */
 ReactDatesConstants.HORIZONTAL_ORIENTATION;
 
+/** @type {!function(?IArrayLike):!Array} */
+Array.prototype.from;
```

Comment:
Is this necessary?

Reply:
It complained about the reference in the polyfill, but it's unnecessary after moving the polyfill to the bundle